2022-02-24 22:16:51,019 INFO    MainThread:25942 [wandb_setup.py:_flush():71] setting env: {}
2022-02-24 22:16:51,019 INFO    MainThread:25942 [wandb_setup.py:_flush():71] setting login settings: {}
2022-02-24 22:16:51,019 INFO    MainThread:25942 [wandb_init.py:_log_setup():371] Logging user logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_221651-fuw4ari5/logs/debug.log
2022-02-24 22:16:51,020 INFO    MainThread:25942 [wandb_init.py:_log_setup():372] Logging internal logs to /home/simon/CDE_UBS/thesis/template/wandb/run-20220224_221651-fuw4ari5/logs/debug-internal.log
2022-02-24 22:16:51,020 INFO    MainThread:25942 [wandb_init.py:_jupyter_setup():321] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x7faee7eab910>
2022-02-24 22:16:51,020 INFO    MainThread:25942 [wandb_init.py:init():404] calling init triggers
2022-02-24 22:16:51,020 INFO    MainThread:25942 [wandb_init.py:init():411] wandb.init called with sweep_config: {}
config: {}
2022-02-24 22:16:51,021 INFO    MainThread:25942 [wandb_init.py:init():460] starting backend
2022-02-24 22:16:51,021 INFO    MainThread:25942 [backend.py:_multiprocessing_setup():101] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-02-24 22:16:51,072 INFO    MainThread:25942 [backend.py:ensure_launched():216] starting backend process...
2022-02-24 22:16:51,111 INFO    MainThread:25942 [backend.py:ensure_launched():222] started backend process with pid: 25987
2022-02-24 22:16:51,113 INFO    MainThread:25942 [wandb_init.py:init():469] backend started and connected
2022-02-24 22:16:51,156 INFO    MainThread:25942 [wandb_run.py:_label_probe_notebook():911] probe notebook
2022-02-24 22:16:51,157 INFO    MainThread:25942 [wandb_run.py:_label_probe_notebook():921] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2022-02-24 22:16:51,157 INFO    MainThread:25942 [wandb_init.py:init():533] updated telemetry
2022-02-24 22:16:51,283 INFO    MainThread:25942 [wandb_init.py:init():563] communicating current version
2022-02-24 22:16:53,254 INFO    MainThread:25942 [wandb_init.py:init():568] got version response upgrade_message: "wandb version 0.12.10 is available!  To upgrade, please run:\n $ pip install wandb --upgrade"

2022-02-24 22:16:53,255 INFO    MainThread:25942 [wandb_init.py:init():578] communicating run to backend with 30 second timeout
2022-02-24 22:16:53,494 INFO    MainThread:25942 [wandb_init.py:init():606] starting run threads in backend
2022-02-24 22:16:58,506 INFO    MainThread:25942 [wandb_run.py:_console_start():1810] atexit reg
2022-02-24 22:16:58,508 INFO    MainThread:25942 [wandb_run.py:_redirect():1684] redirect: SettingsConsole.WRAP
2022-02-24 22:16:58,508 INFO    MainThread:25942 [wandb_run.py:_redirect():1721] Wrapping output streams.
2022-02-24 22:16:58,510 INFO    MainThread:25942 [wandb_run.py:_redirect():1745] Redirects installed.
2022-02-24 22:16:58,510 INFO    MainThread:25942 [wandb_init.py:init():633] run started, returning control to user process
2022-02-24 22:17:09,097 INFO    MainThread:25942 [wandb_init.py:_pause_backend():284] pausing backend
2022-02-24 22:17:09,098 INFO    MainThread:25942 [jupyter.py:save_ipynb():374] not saving jupyter notebook
2022-02-24 22:18:14,246 INFO    MainThread:25942 [wandb_run.py:_atexit_cleanup():1780] got exitcode: 0
2022-02-24 22:18:14,248 INFO    MainThread:25942 [wandb_run.py:_restore():1752] restore
2022-02-24 22:18:15,125 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 22:18:15,333 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 1
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 761
}

2022-02-24 22:18:15,552 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 27281
}

2022-02-24 22:18:15,660 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 27281
}

2022-02-24 22:18:15,764 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 761
  total_bytes: 27281
}

2022-02-24 22:18:15,868 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 2795
  total_bytes: 27281
}

2022-02-24 22:18:15,977 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27281
  total_bytes: 27281
}

2022-02-24 22:18:16,082 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27281
  total_bytes: 27281
}

2022-02-24 22:18:16,191 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27281
  total_bytes: 27281
}

2022-02-24 22:18:16,294 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27281
  total_bytes: 27281
}

2022-02-24 22:18:16,398 INFO    MainThread:25942 [wandb_run.py:_wait_for_finish():1912] got exit ret: file_counts {
  wandb_count: 6
}
pusher_stats {
  uploaded_bytes: 27281
  total_bytes: 27281
}

